dist/config/routes/monsieurbiz_sylius_rich_editor_plugin.yaml

```diff
@@ -0,0 +1,3 @@
+monsieurbiz_richeditor_admin:
```

Comment:
Symlink? :p

Reply:
It is but at the end, we will not have recipes files anymore when we will have the recipes on Symfony side.